source/faq.txt

```diff
@@ -261,31 +261,58 @@ socket connections.
 If you experience unexpected network behavior, here
 are some things to check:
 
-#. The firewall should send a ``FIN packet`` when closing a socket,allowing the driver to detect that the socket is closed.
+#. The firewall should send a ``FIN packet`` when closing a socket,
+   allowing the driver to detect that the socket is closed.
 #. The firewall should allow ``keepAlive`` probes.
 
 How Can I Prevent a Slow Operation From Delaying Other Operations?
 ------------------------------------------------------------------
 
-To control the maximum size of a connection pool, you can set the
-``maxPoolSize`` option in the :ref:`connection options
-<node-connection-options>`. The default value of ``maxPoolSize`` is
-``100``. If the number of in-use connections to a server reaches
-``maxPoolSize``, the next request to that server will wait 
-until a connection becomes available. To prevent long-running operations
-from slowing down your application, you can increase ``maxPoolSize``.
-
-The driver does not limit the number of requests that can wait for
-sockets to become available. Requests wait for the amount of time
-specified in the ``waitQueueTimeoutMS`` option, which
-defaults to ``0`` (no limit). You should set this option if it is
-more important to stop long-running operations than it is to complete
-every operation.
+When you use the same ``MongoClient`` instance to run multiple MongoDB
+operations concurrently, a slow operation can cause delays to other
+operations. The slow operation keeps a connection to MongoDB occupied,
```

Comment:
Suggestion:
I think this should describe the general case since it's not referring to a specific one in the previous sentence.

```suggestion
operations. Slow operations keep a connection to MongoDB occupied,
```